pages/code_of_conduct.vue

```diff
@@ -196,30 +196,70 @@
                     6. Reporting Guidelines
                 </h2>
                 <p class="py-2">
-                    If you are subject to or witness unacceptable behavior, or
-                    have any other concerns, please notify a community organizer
-                    as soon as possible.
+                    Largely taken from
+                    <a
+                        class="text-brand-primary"
+                        href="https://www.djangoproject.com/conduct/reporting/"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        Django
+                    </a>
+                    and
+                    <a
+                        class="text-brand-primary"
+                        href="https://otter.technology/"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        >Otter Tech</a
+                    >. Thanks!
                 </p>
                 <p class="py-2">
+                    If you are subject to or witness unacceptable behavior
+                    please notify the code of conduct committee by filling out
+                    the form below.
+                </p>
+                <form action="https://forms.gle/NR56fRQUVzdbf1v3A">
+                    <button
+                        type="submit"
+                        class="inline-flex justify-center py-2 px-4 border border-transparent text-sm leading-5 font-medium rounded-md text-white bg-indigo-600 hover:bg-indigo-500 focus:outline-none focus:border-indigo-700 focus:shadow-outline-indigo active:bg-indigo-700 transition duration-150 ease-in-out my-5"
+                    >
+                        Report a code of conduct violation
+                    </button>
+                </form>
+                <p class="py-2">
+                    If you have any questions please email
                     <a
                         class="text-brand-primary"
-                        href="mailto:coc-report@devedmonton.com"
-                    >
-                        coc-report@devedmonton.com</a
+                        href="mailto:board@devedmonton.com"
                     >
+                        board@devedmonton.com</a
```

Comment:
Is this still the right email for CoC violations?

Reply:
I think so. 

Eventually we would want it going to the committee but given that that hasn't been formed yet and we haven't created an email "board" is better than the coc one cause I think the coc one just going to you currently